tests/src/resultcollection.hh

```diff
@@ -123,6 +201,36 @@ inline auto linearVonMisesResultsOfCube = []<typename NOP, typename FE>(NOP& non
   return std::make_tuple(feRequirements, expectedStress, Ikarus::utils::referenceElementVertexPositions(fe));
 };
 
+inline auto linearTriaxialityResultsOfCube = []<typename NOP, typename FE>(NOP& nonLinearOperator, FE& fe) {
+  constexpr int vertices   = 8;
+  constexpr int quantities = 1;
+
+  const Eigen::Matrix<double, vertices, quantities> expectedStress{0.68516016, NaN(), -0.40946151, -0.68516016,
+                                                                   NaN(),      NaN(), -0.68516016, NaN()};
+
+  auto& displacement = nonLinearOperator.firstParameter();
+  displacement << 0, 0, 0, 0, 0, 0, 1, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0;
+
+  auto feRequirements = typename FE::Requirement().insertGlobalSolution(displacement);
+
+  return std::make_tuple(feRequirements, expectedStress, Ikarus::utils::referenceElementVertexPositions(fe));
+};
+
+inline auto linearHydrostaticStressResultsOfCube = []<typename NOP, typename FE>(NOP& nonLinearOperator, FE& fe) {
+  constexpr int vertices   = 8;
+  constexpr int quantities = 1;
+
+  const Eigen::Matrix<double, vertices, quantities> expectedStress{833.33333333,  0, -833.33333333, -833.33333333, 0, 0,
+                                                                   -833.33333333, 0};
+
+  auto& displacement = nonLinearOperator.firstParameter();
+  displacement << 0, 0, 0, 0, 0, 0, 1, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0;
```

Comment:
I this test collection you have a lot of duplication of `displacement <<`.
It would also be more nice to document what deformation that is.
Therefore, maybe we can have at the top where the NaN is:

```cpp

Eigen::VectorXd q1PureShearDisplacement({....});
Eigen::VectorXd q1UniAxialStrectchXDisplacement({....});
Eigen::VectorXd h1UniAxialStrectchXDisplacement({....});

```

or for shorter names
```cpp
struct Q1Displacements
{
static Eigen::VectorXd pureShearDisplacement({....});
static Eigen::VectorXd oniAxialStrectchXDisplacement({....});
};
```

Then only the results have to be written.